src/Libraries/CoreNodeModels/DefineData.cs

```diff
@@ -9,28 +8,51 @@
 using Newtonsoft.Json;
 using ProtoCore.AST.AssociativeAST;
 using VMDataBridge;
+using static DSCore.Data;
 
 
 namespace CoreNodeModels
 {
     [NodeName("DefineData")]
     [NodeDescription(nameof(Properties.Resources.RememberDescription), typeof(Properties.Resources))]
     [NodeCategory("Core.Data")]
-    [InPortNames(">")]
-    [InPortTypes("var[]..[]")]
-    [InPortDescriptions(typeof(Properties.Resources),
-        nameof(Properties.Resources.RememberInputToolTip))]
     [OutPortNames(">")]
     [OutPortTypes("var[]..[]")]
     [OutPortDescriptions(typeof(Properties.Resources), nameof(Properties.Resources.RememberOuputToolTip))]
     [IsDesignScriptCompatible]
     public class DefineData : DSDropDownBase
     {
-        private bool context;
         private List<DynamoDropDownItem> serializedItems;
+        private bool isAutoMode;
+        private bool isList;
 
+        /// <summary>
+        /// AutoMode property
```

Comment:
comments would be great, I am pretty sure no one seeing this method for the first time will have any idea what AutoMode does.

Reply:
Absolutely, it is quite confusing.